src/table/cells/handle-cell-focus.js

```diff
@@ -89,6 +89,57 @@ export const handleNavigateTop = ({ tableSectionRef, focusedCellCoord, rootEleme
   }
 };
 
+export const getCellSelectionStatusNote = (rows, translator) =>
+  rows.length === 1
+    ? translator.get('SNTable.SelectionLabel.OneSelectedValue')
+    : translator.get('SNTable.SelectionLabel.SelectedValues', [rows.length]);
+
+export const getMemoisedSrNotation = (prevCount = 0) => {
```

Comment:
How is this Memoised?

Reply:
I'm passing the `prevCount` value here only for testing purpose, in test cases we need to increase this value in order to be test different cases. in the code side we never increase it by passing it to the function.